bin.js

```diff
@@ -0,0 +1,11 @@
+const { resolve, join } = require("path");
```

Comment:
I think bin.js and bin.d.ts would better in src/*, structure-wise, given that bin.js is just called by src/index.ts, right ? 

Also .ts and es6 import / export instead, to keep consistency

Reply:
`bin.js` is used by `install.js` also which is why I pulled it out into in l it's own file. But, `install.js` could reference the built `bin.ts` so I will move it to `src`.

Reply:
@Ribeiro-Tiago Actually, I think it might be better to keep `bin.js` out of TypeScript so that it doesn't have to be built to be used. 

If you clone the repo and run `npm i` or `yarn`, `install.js` will fail since the TS isn't built yet. I could add a check to see if the TS files are built yet and run `install.js` or not based on that. But in my opinion, it would be better to just keep `bin.js` as a regular JS file so that `install.js` always works. 

Thoughts?

Reply:
I agree, but I don't see how bin.js is used by install.js, what am I missing ?

Reply:
I renamed `bin.js` to `constants.js` so that the purpose of the file is more clear. See lines 4-8 in `install.js` for where it is used.

https://github.com/eventOneHQ/bundletool/blob/35250887e5e08cd03dac84473636984dbf29282e/install.js#L4-8

Reply:
Right, my apologies